requirements.txt

```diff
@@ -23,4 +23,6 @@ numba
 emoji
 grapheme
 psutil
-lolcat
\ No newline at end of file
+lolcat
```

Comment:
curious -- why are we storing unversioned requirements? Python projects generally only store versioned deps, which can be updated manually if desired

Reply:
When searching how to GREP only the requirements in the project, each forum/tutorial I came across had 2 requirements, one with versions and one without. Since every thread I read had that file present, I assumed it was best practice. Other than that, there is no reason for keeping an additional requirements.txt

Reply:
Removed unversioned requirements